learning_assistant/views.py

```diff
@@ -149,3 +154,69 @@ def get(self, request, course_run_id):
         }
 
         return Response(status=http_status.HTTP_200_OK, data=data)
+
+
+class LearningAssistantMessageHistoryView(APIView):
+    """
+    View to retrieve the message history for user in a course.
+
+    This endpoint returns the message history stored in the LearningAssistantMessage table in a course
+    represented by the course_key, which is provided in the URL.
+
+    Accepts: [GET]
+
+    Path: /learning_assistant/v1/course_id/{course_key}/history/{message_count}
+
+    Parameters:
+        * course_key: the ID of the course
+
+    Responses:
+        * 200: OK
+        * 400: Malformed Request - Course ID is not a valid course ID.
+    """
+
+    authentication_classes = (SessionAuthentication, JwtAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, course_run_id, message_count=50):
```

Comment:
I still think this should be a query parameter, not a path parameter.

Reply:
I keep mixing these up- I think I've got it right now, gonna use `request.GET.get('message_count', 50) instead.

Side note, what do you think the default value should be? Is 50 good?

Reply:
Yes, that should be the right syntax! And 50 seems good.